### ISSUE TITLE
darwin arm64 build?

### ISSUE THREAD

**Issue by sungo** [EdJoPaTo/mqttui#4]
I love this app and use it almost daily. I just got a new M1 mac though and I'd love a darwin arm64 build. I have no idea if this is an onerous request though so I will not be offended if the answer is no :)

**Comment by EdJoPaTo** [comment]
I like to hear that! :)

Regarding the M1 / apple arm64 build: I saw some GitHub Actions attempts on building for the M1 architecture. Some attempted it with the not yet available macOS-11 GitHub action runners, some used the current macOS-latest ones with a specific variable set.
I can take a look into that, maybe it works out fairly easy.

You can definitely compile the binary yourself as an arm64 build.
Assuming you have homebrew installed:
```sh
brew install rustup-init
rustup default stable
git clone https://github.com/EdJoPaTo/mqttui.git
cd mqttui
cargo install --path .
```

the final binary is then placed into ~/.cargo/bin/mqttui

**Comment by sungo** [comment]
Excellent. I installed rust and I can at least report that mqttui builds and runs cleanly on darwin aarch64 (as it's known to rust).

**Comment by EdJoPaTo** [comment]
I added the aarch64-apple-darwin build in 0b464eaa93fab3b74143e3b02cdccb28278e195d and released [v0.11.1](https://github.com/EdJoPaTo/mqttui/releases/tag/v0.11.1) which will now include a build for Apple M1 / arm64 darwin.

Thanks for bringing this up!
